src/auth-service/routes/api.js

```diff
@@ -40,8 +40,8 @@ router.post("/forgotPassword", joinController.forgotPassword);
 router.get("/findUser", joinController.findUser);
 router.put("/", joinController.updateUser);
 router.delete("/:id", joinController.deleteUser);
-router.put("/defaults/:id", joinController.updateUserDefaults);
-router.get("/defaults/:id", joinController.getDefaults);
+router.put("/defaults", joinController.updateUserDefaults);
```

Comment:
Could we add the record/chart id in the path rather than in the body
```suggestion
router.put("/defaults/:id", joinController.updateUserDefaults);
```

Reply:
Okay, sounds good. Let me address the suggestion asap.

Reply:
> Could we add the record/chart id in the path rather than in the body

This has been addressed using query params as seen in the updated PR description. Please let me know if this works fine on your end.